CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+## How to contribute
+
+If you are interested in joining the project, then please review our [Projects Board](https://github.com/hackforla/VRMS/projects/2) to see if anything interests you. If so, then please do the following.
+
+1. Review the Hack for LA Getting Started Guide https://www.hackforla.org/getting-started.
+
+1. Join the VRMS weekly meetings https://www.vrms.io/.
+
+1. Reach out to our project leads to discuss how you would like to contribute.
+
+### Committing changes overview
+
+1. Find a ticket in the [Prioritized Backlog](https://github.com/hackforla/VRMS/projects/2) and assign yourself to the ticket.
```

Comment:
and assign yourself to the ticket. => , assign yourself to the ticket, and update its status as "In progress".

Reply:
👍